nexus_constructor/validators.py

```diff
@@ -32,27 +32,25 @@ def validate(self, input: str, pos: int):
             AttributeError,
             pint.compat.tokenize.TokenError,
         ):
-            self.validationFailed.emit()
+            self.isValid.emit(False)
             return QValidator.Intermediate
 
         # Attempt to find 1 metre in terms of the unit. This will ensure that it's a length.
         try:
             self.ureg.metre.from_(unit)
         except (pint.errors.DimensionalityError, ValueError):
-            self.validationFailed.emit()
+            self.isValid.emit(False)
             return QValidator.Intermediate
 
         # Reject input in the form of "2 metres," "40 cm," etc
         if unit.magnitude != 1:
-            self.validationFailed.emit()
+            self.isValid.emit(False)
             return QValidator.Intermediate
 
-        self.validationSuccess.emit()
+        self.isValid.emit(True)
         return QValidator.Acceptable
 
-    # Create signals because the QML LabeledTextField doesn't have "onAccepted"
-    validationSuccess = Signal()
-    validationFailed = Signal()
+    isValid = Signal(bool)
```

Comment:
Is this a standard signal name which must be camelCase, or can we use snake_case?

Reply:
we can use snake_case here i believe.